db/data/20220331152016_fix_lead_school_not_applicable_for_hesa_records.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class FixLeadSchoolNotApplicableForHesaRecords < ActiveRecord::Migration[6.1]
+  def up
+    Trainee.imported_from_hesa.where(lead_school_id: nil).where.not(employing_school_id: nil).each do |trainee|
```

Comment:
Will this exclude School Direct (fee funded) that might have a lead school that is not applicable but won't have an employing school?